src/fast_check/transform.rs

```diff
@@ -110,8 +110,10 @@ impl CommentsMut {
       comments.retain(|_key, value| {
         value.retain(|c| {
           match c.kind {
-            // only keep js docs and @deno-types comments
-            CommentKind::Line => c.text.starts_with("@deno-types"),
+            // only keep js docs, @ts-*, and @deno-types comments
+            CommentKind::Line => {
+              c.text.starts_with("@deno-types") || c.text.starts_with("@ts-")
```

Comment:
Oh, this actually might need to use `.trim_start()` first for both of these I think.

Reply:
We should definitely have a test for that.